### ISSUE TITLE
Prevent log files from being generated in project folder?

### ISSUE THREAD

**Issue by borice** [dragos/dragos-vscode-scala#97]
I'm noticing that this extension, when enabled in VSCode, creates two files in the current project folder:

> ensime-langserver.log
> pc.stdout.log

Any way to prevent these files from being generated? (or at least have them be generated in some temp location)?

My environment:
VSCode 1.32.3
Scala Language Server 0.2.3

My Scala project uses sbt 1.2.8 with scala-2.12.8 and sbt-ensime 2.5.1  (i tried 2.6.1 also but same result).  OSX 10.14.3   Java 8

**Comment by dragos** [comment]
Nope, there's no way currently. I'm happy to merge a PR to move them somewhere else though, but I don't have time to work on this in the near future.
